assets/stylesheets/about.css

```diff
@@ -0,0 +1,339 @@
+#about-hero {
+  margin-top: 40px;
+  height: 100%;
+  background-image: url('../../assets/images/about-hero.png');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  padding: 0;
+  display: grid;
+  grid-template-rows: 1.5fr 1.5fr 1fr;
+}
+
+#about-hero-bar {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
[OPTIONAL] This line repeats many times in your CSS files. Kindly follow the mobile-first approach and use the desktop media query once in a file to make your code readable and DRY.